packages/kitsu-core/src/serialise/index.spec.js

```diff
@@ -181,6 +181,21 @@ describe('kitsu-core', () => {
       })
     })
 
+    it('does not pluralise type', () => {
+      expect.assertions(1)
+      const input = serial.nonePlural('AttackOnTitan', {
```

Comment:
Can this test use the same input as the `it pluralises type` test for consistency? https://github.com/wopian/kitsu/pull/395/files#diff-7f9515c849c02abac7347c56621812bcR154